auto_submit/bin/server.dart

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+
+import 'package:auto_submit/helpers.dart';
+import 'package:googleapis/firestore/v1.dart';
+import 'package:googleapis_auth/auth_io.dart';
+import 'package:shelf/shelf.dart';
+import 'package:shelf_router/shelf_router.dart';
+
+Future main() async {
+  final projectId = await currentProjectId();
+  print('Current GCP project id: $projectId');
+
+  final authClient = await clientViaApplicationDefaultCredentials(
+    scopes: [FirestoreApi.datastoreScope],
+  );
+
+  try {
+    final api = FirestoreApi(authClient);
```

Comment:
Could you please change the example API to a different one, like the BigqueryApi? Our current project does not support Cloud Firestore API (Below is the error I hit when running locally). This would make it easier to avoid any local run errors for @KristinBi to start with.
```
Error thrown by handler.
DetailedApiRequestError(status: 400, message: The Cloud Firestore API is not available for Datastore Mode projects.)
```

Reply:
I removed the firestore project. I'll let @KristinBi start with an empty codebase and decide what dependencies we should pull in

Reply:
SG. Thanks!